src/shared/codelens/pythonDebugAdapterHeartbeat.ts

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as net from 'net'
+import { getLogger, Logger } from '../logger'
+
+const DEBUG_ADAPTER_RESPONSE_TIMEOUT_MILLIS = 1500
+
+export class PythonDebugAdapterHeartbeat {
+    private readonly socket: net.Socket
+    private readonly logger: Logger = getLogger()
+    public constructor(private readonly port: number) {
+        this.socket = new net.Socket()
```

Comment:
we should be able to `import { Socket }` since it's the only thing used from the package